Makefile.js

```diff
@@ -1088,7 +1088,7 @@ target.perf = function() {
                 // Count test target files.
                 const count = glob.sync(
                     process.platform === "win32"
-                        ? PERF_MULTIFILES_TARGETS.slice(2).replace("\\", "/")
+                        ? PERF_MULTIFILES_TARGETS.slice(2).replace(/\/\//gu, "/")
```

Comment:
Should this be `/\\/gu/`?

Reply:
oh, thanks for catching this, I've fixed it.